source/common/http/match_delegate/config.cc

```diff
@@ -285,15 +285,15 @@ Envoy::Http::FilterFactoryCb MatchWrapperConfig::createFilterFactoryFromProtoTyp
 }
 
 /**
- * Static registration for the match wrapper filter. @see RegisterFactory.
- * Note that we register this as a filter in order to serve as a drop in wrapper for other HTTP
+ * Static registration for the match delegate filter. @see RegisterFactory.
+ * Note that we register this as a filter in order to serve as a drop in delegate for other HTTP
  * filters. While not a real filter, by being registered as one all the code paths that look up
  * HTTP filters will look up this filter factory instead, which does the work to create and
  * associate a match tree with the underlying filter.
```

Comment:
At this point we're treating it just as any other filter, so maybe update the text here?

Reply:
Get it. Thanks.